docs/Public_Chains.md

```diff
@@ -0,0 +1,61 @@
+---
+id: Public_Chains
+title: Public Arbitrum Chains
+sidebar_label: Public Arbitrum Chains
+---
+
+The following is a comprehensive list of all of the currently live Arbitrum chains:
+
+
+
+| Name                                     | Chain ID | Type    | Underlying L1 | Current Tech Stack | RPC Url(s)                                                                                                                                     | Explorer(s)                                                        | Native Currency | Retryable Dashboard                          |
+| ---------------------------------------- | -------- | ------- | ------------- | ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ | --------------- | -------------------------------------------- |
+| Arbitrum One                             | 42161    | Mainnet | Ethereum      | Classic Rollup     | https://arb1.arbitrum.io/rpc <br> https://arbitrum-mainnet.infura.io/v3/YOUR-PROJECT-ID <br> https://arb-mainnet.g.alchemy.com/v2/your-api-key | https://arbiscan.io/ https://explorer.arbitrum.io/                 | ETH             | https://retryable-dashboard.arbitrum.io/     |
+| Arbitrum Nova                            | 42170    | Mainnet | Ethereum      | Nitro AnyTrust     | [coming soon!]                                                                                                                                 | https://a4ba-explorer.arbitrum.io/                                 | ETH             | http://retryable-tx-panel-nitro.arbitrum.io/ |
```

Comment:
I think we're ok to start using the new tld names:
Explorer: https://nova-explorer.arbitrum.io/
RPC URL: https://nova.arbitrum.io/rpc <-- we're already sending this to teams so it should be ok to share publicly?

Reply:
for some reason it wouldn't let me issue a suggestion here